src/Backend/Executor.ts

```diff
@@ -18,6 +18,9 @@ import {Command} from './Command';
 import {Toolchains} from './Toolchain';
 
 interface Executor {
+  // exetensions of executable files
+  getExecutableExt(): string[];
```

Comment:
Like ['tflite', 'onxx']?

Reply:
Exactly. IMO, After one-infer implementaion, every ONE toolchain might have tflite and onnx as its exacutable extension.